Makefile

```diff
@@ -1,2 +1,12 @@
+export REPOSITORY=flytectl
+include boilerplate/lyft/golang_test_targets/Makefile
+
 generate:
-	go test github.com/lyft/flytectl/cmd --update
\ No newline at end of file
+	go test github.com/lyft/flytectl/cmd --update
+
+compile:
```

Comment:
what's the difference between this? https://github.com/lyft/boilerplate/tree/master/boilerplate/lyft/flyte_golang_compile

Reply:
not much, can change, i just added this for quick iteration